model/project.go

```diff
@@ -941,12 +941,30 @@ func (p *Project) BuildProjectTVPairsWithAlias(alias string) ([]TVPair, error) {
 		for _, variant := range p.BuildVariants {
 			if variantRegex.MatchString(variant.Name) {
 				for _, task := range p.Tasks {
-					if ((v.Task != "" && taskRegex.MatchString(task.Name)) ||
-						(len(v.Tags) > 0 && len(util.StringSliceIntersection(task.Tags, v.Tags)) > 0)) &&
-						(p.FindTaskForVariant(task.Name, variant.Name) != nil) {
+					if task.Patchable != nil && !(*task.Patchable) {
+						continue
+					}
+					if !((v.Task != "" && taskRegex.MatchString(task.Name)) ||
```

Comment:
this ! should be inside the parens I think?

Reply:
Nope, I'm negating the old condition, so it's in the right place.